python/covid_web/covid_web/server.py

```diff
@@ -32,6 +34,24 @@ def run_server(data_dir: Path, prediction_dir: Path) -> None:
     server.run(host="0.0.0.0", port=8081)
 
 
+GA_TRACKING_ID = os.environ["GA_TRACKING_ID"]
+
+
+def track_pageview(path):
+    data = {
+        "v": "1",
+        "tid": GA_TRACKING_ID,  # Tracking ID / Property ID.
+        "cid": 47,
+        "t": "pageview",
+        "dp": f"/covid19/{path}",
+        "dh": "graphs.lukipuki.sk",
+        "ua": request.headers.get("User-Agent"),
+    }
+
+    response = requests.post("https://www.google-analytics.com/collect", data=data)
+    response.raise_for_status()
```

Comment:
Do we really need to add also a python code for tracking? Isn't included JS in HTML enough?

Reply:
If I were to choose, I'd choose the Python part and drop JS from `index.html`. The `index.html` script is only for the root page, which no one visits. I put it there mostly to test the functionality, as the Python version wasn't working.